src/app/client/page.tsx

```diff
@@ -30,8 +31,8 @@ async function getAllTasks(token: string): Promise<TaskResponse[]> {
 }
 
 async function getAssigneeList(token: string): Promise<IAssignee> {
-  const res = await fetch(`${apiUrl}/api/users/client?token=${token}`, {
-    next: { tags: ['getAssigneeList'] },
+  const res = await fetch(`${apiUrl}/api/users/client?token=${token}&limit=${MAX_FETCH_ASSIGNEE_COUNT}`, {
+    next: { tags: ['getAssigneeList'], revalidate: ASSIGNEE_REVALIDATION_INTERVAL },
   })
```

Comment:
This revalidation will be overridden by `export const fetchCache = 'force-no-store'` so it can be removed.